vividus-tests/src/main/resources/known-issues.json

```diff
@@ -93,7 +93,7 @@
   },
   "VVD-14": {
     "type": "Internal",
-    "assertionPattern": "Jump link by selector .* points to missing element with notFound id",
+    "assertionPattern": "(Jump link by selector .* points to missing element with notFound id or name|Element by selector .* doesn't contain href/src attributes)",
```

Comment:
why is the second part of the pattern needed?

Reply:
> why is the second part of the pattern needed?

Now we have [named anchor](https://www.tagindex.net/html/link/a_name.html) on test page
https://vividus-test-site-a92k.onrender.com/links.html#ElementName

Hmmm, it would probably be better if we used locators that exclude such links